README.md

```diff
@@ -21,6 +21,10 @@ resource "aws_instance" "foo" {
 
 Since `t1.2xlarge` is a nonexistent instance type, an error will occur when you run `terraform apply`. But `terraform plan` and `terraform validate` cannot find this possible error beforehand. That's because it's an AWS provider-specific issue and it's valid as a Terraform configuration.
 
+**P.S.**
+TFLint always checks only the current root module (no recursive check).
+For more info please see: https://github.com/wata727/tflint/issues/493 and https://github.com/wata727/tflint/issues/502
+
```

Comment:
I think it would be nice to add FAQ section like at the bottom. What do you think?

Reply:
Okay I will update it then with my question. Let me know if you want to add any more questions.

Reply:
Thanks! In that case, I think that we don't need to add this line.